applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp

```diff
@@ -1258,7 +1241,7 @@ namespace Kratos
 			Stress = ZeroVector(msElementSize);

```

Comment:
Why do you assign twice?, the cost of filling with zeros is the same as filling with any other double, if you are going to do += is fine, because that way you initialize the values, but as you use = your are doing the same twice

Reply:
are you referring to "Stress"? I will change this, thanks